### DISCUSSION TITLE
字幕显示的疑问

### DISCUSSION THREAD

**Discussion by linger040831** [hooke007/mpv_PlayKit#237] [mpv 通用]
这是win上mpv的显示

<img width="962" alt="屏幕截图_20221126_211506" src="https://user-images.githubusercontent.com/84896599/204091003-e7da8c5d-e91c-4198-83dc-3aeca34bf021.png">

这是手机上reex的显示

![Screenshot_20221126-211610](https://user-images.githubusercontent.com/84896599/204091017-dfac208c-032b-4247-ac53-64d00a9d039a.png)

 这是win上的字幕配置

<img width="456" alt="屏幕截图_20221126_211653" src="https://user-images.githubusercontent.com/84896599/204091030-bdbe6d6d-e3ab-4f95-9024-ccfa40682689.png">

这是手机上的

![Screenshot_20221126-211620](https://user-images.githubusercontent.com/84896599/204091037-28ec19a6-97fd-460f-8019-2a2b6ad8f3b8.png)

手机和win都开了 blend-subtitles = video

手机的字幕显示起来就是正常的，win上的就会有一点重叠，win上用字幕缩放就可以正常显示

<img width="962" alt="屏幕截图_20221126_211536" src="https://user-images.githubusercontent.com/84896599/204091102-71f8242f-e28b-4e4c-9d35-aa664f9ae58a.png">

请问这是为什么，手机和win的字体库是一样的，是字体的原因造成的吗？

**Comment by hooke007** [comment]
你虽然写了很多。。。但是控制变量的时候，最基本的使用同一字体却没做到。。。。
